src/Doctrine/ORM/PixelRepository.php

```diff
@@ -4,20 +4,28 @@
 
 namespace Setono\SyliusFacebookPlugin\Doctrine\ORM;
 
+use Setono\SyliusFacebookPlugin\Model\PixelInterface;
 use Setono\SyliusFacebookPlugin\Repository\PixelRepositoryInterface;
 use Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository;
 use Sylius\Component\Channel\Model\ChannelInterface;
+use Webmozart\Assert\Assert;
 
 class PixelRepository extends EntityRepository implements PixelRepositoryInterface
 {
     public function findEnabledByChannel(ChannelInterface $channel): array
     {
-        return $this->createQueryBuilder('o')
+        /** @psalm-suppress QueryBuilderSetParameter */
```

Comment:
Suppress this globally: https://github.com/Setono/SyliusAnalyticsPlugin/blob/2.0.x/psalm.xml#L20